great_tables/_formats.py

```diff
@@ -1328,17 +1335,30 @@ def fmt_currency_context(
 
     # Format the value to decimal notation; this is done before the currency symbol is
     # affixed to the value
-    x_formatted = _value_to_decimal_notation(
-        value=x,
-        decimals=decimals,
-        n_sigfig=None,
-        drop_trailing_zeros=False,
-        drop_trailing_dec_mark=drop_trailing_dec_mark,
-        use_seps=use_seps,
-        sep_mark=sep_mark,
-        dec_mark=dec_mark,
-        force_sign=force_sign,
-    )
+    if compact:
```

Comment:
I think you could define the function to use in the if/else block, and then call it after, eg...

```python
if compact:
    f_formatter = _format_number_compactly
else:
    f_formatter = _value_to_decimal_notation

x_formatted = f_formatter(...)
```

Reply:
This has now been changed. Thanks!